src/browser.ts

```diff
@@ -116,6 +116,27 @@ function flushAnonymousUser(analytics: Analytics): void {
   }
 }
 
+/**
+ * Ensures events can be registered before library initialization.
+ * This is important so users can register to 'initialize' and any events that may fire early during setup.
+ */
+
+function flushEventEmitters(analytics: Analytics): void {
```

Comment:
Can we merge the `flushAnonymousUser` and `flushEventEmitters` into one function to avoid repeating the shared logic?